cascade/src/main/java/me/saket/cascade/ktx.kt

```diff
@@ -34,18 +34,30 @@ fun Menu.add(
   title: CharSequence,
   itemId: Int = Menu.NONE,
   groupId: Int = Menu.NONE,
-  order: Int = Menu.NONE
-): MenuItem {
-  return add(groupId, itemId, order, title)
+  order: Int = Menu.NONE,
+  menuItemClickListener: ((MenuItem) -> Unit)? = null
```

Comment:
Let's rename this to `onClick`?

Reply:
d08423b6c4f461df63bef7e4d8699b20e2242f47